<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QColorToolClass</class>
 <widget class="QWidget" name="QColorToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>436</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>QColorTool</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>文件</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="toolButton_open">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_save">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>选择</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton_select_new">
          <property name="text">
           <string>new</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButton_select_union">
          <property name="text">
           <string>union</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButton_select_inter">
          <property name="text">
           <string>intersect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="toolButton_select_diff">
          <property name="text">
           <string>diff</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>编辑</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButton_color">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton_delete">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>点云简化</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>网格尺寸：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_simple">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_simple">
          <property name="text">
           <string>应用</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRendView" name="view" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QRendView</class>
   <extends>QWidget</extends>
   <header>qrendview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qcolortool.qrc"/>
 </resources>
 <connections/>
</ui>
